File: Tarea07/ejercicio4.py
f = [0,0,0,0,0,0,0,0,0,0]
print("Ingresa 10 numeros")
for n in range(0, 10):
	f[n] = int(input())
print("Ingresa el numero de lugares desplazados")
desp = int(input())
print("Shidt a la izquierda")
for n in range(0, 10):
	a = (desp + n) % 10
	print("%i, "%(f[a]), end = "")
print("\nShift a la derecha\n")
for n in range(0, 10):
	a = (n-desp+10) % 10
	print("%i, "%(f[a]), end="")
print("\n")

File: Tarea07/ejercicio4.c
#include <stdio.h>

int main(){
	int n, f[10], desp, a;
	printf("Ingrsa 10 numeros\n");
	for (int n = 0; n < 10; ++n){
		scanf("%d", &f[n]);
	}
	printf("Ingresa el numero de lugares desplazados\n");
	scanf("%d", &desp);
	printf("Shift a la izquierda\n");
	for (n = 0; n < 10; ++n){
		a = (desp + n) % 10;
		printf("%i, ", f[a]);
	}
	printf("\nShift a la derecha\n");
	for (n = 0; n < 10; ++n){
		a = (n-desp+10) % 10;
		printf("%i, ", f[a]);
	}
	printf("\n");
	return 0;
}